crates/hyfetch/build.rs

```diff
@@ -147,6 +147,10 @@ impl Distro {
 
     let quotes = "#".repeat(80);
     for (variant, AsciiDistro { art, .. }) in &variants {
+        // Unescape backslashes here because backslashes are escaped in neofetch
+        // for printf
+        let art = art.replace(r"\\", r"\");
```

Comment:
I think we can do this earlier, when we grab the ascii art from neofetch script.

Reply:
Yeah that's true. This location I had was the equivalent location in the Python version, but that's definitely a better location. Not a big deal to diverge from the reference Python for this kind of improvement.

Fixed.